### ISSUE TITLE
Aws issue

### ISSUE THREAD

**Issue by naveenbabusa** [switnet-ltd/quick-jibri-installer#9]
One issue is in Aws the bash is not working it fails in kernel issue and mod loop asna fails . I tried in gcp it got installed not sure it had few challenges as it kept restarting the meeting once you start the recording and also when we did streaming . Also video was not getting streamed . Any guidance it seem to be unstable . Also grapans I was not able to access

**Comment by Ark74** [comment]
AWS is a very particular case, and indeed we have seen that behaviour.

The real issue is that AWS controls what kernel is loaded, and AFAICT, it also depends on what instance tier you are running, so even when we install the right driver and kernel, aws will load their kernel which lacks snd-aloop, you might wanna manually edit grub in order to set the correct kernel for this to work.

TLDR; aws  take over on what kernel your instance run, thus is requires manual grub edit.

On the second part of this issue, Jibri is one of the services that you want to have plenty resources since it has a pretty quick usage escalation curve, not as fast as resources escalation.
I've seen that under powered servers tend to encounter more issues when running recordings or streamings.

We have tested our installer with all the current *buntu LTS versions so far working fine.
I'd help your report to share some more details on what instance type, server OS, resources, kernel, jibri version your are running otherwise you'll be on your own.

Regards

**Comment by naveenbabusa** [comment]
Thanks for the feedback I did it manually and rest was installed with your bash command . It worked perfectly . Thanks
